UnityWeld/Binding/Internal/TypeResolver.cs

```diff
@@ -101,6 +147,21 @@ private static Type[] FindTypesMarkedByAttribute(Type attributeType)
             return typesFound.ToArray();
         }
 
+        /// <summary>
+        /// Find all types marked with binding attribute
+        /// </summary>
+        /// <returns></returns>
+        private static Type[] FindTypesMarkedByBindingAttribute()
+        {
+            var result = new List<Type>();
+            foreach (var attributeType in BindingAttributeTypes)
+            {
+                result.AddRange(FindTypesMarkedByAttribute(attributeType));
+            }
+
+            return result.ToArray();

```

Comment:
This could be simplified to `return BindingAttributeTypes.SelectMany(FindTypesMarkedByAttribute)`. Not sure what the performance difference would be but here we are already allocating a new list and array, so using that would let you just go straght `ToArray()` without the intermediate step. Potentially returning the IEnumerable from SelectMany could have some advantages due to being lazy.